xilem_core/src/sequence.rs

```diff
@@ -1,363 +1,650 @@
-// Copyright 2023 the Xilem Authors
+// Copyright 2024 the Xilem Authors
```

Comment:
Not sure about these year changes; they're mostly accidental

Reply:
I mean most of the code here is rewritten, so 2024 makes sense? But maybe we also want to have year-ranges to be more accurate.